.github/workflows/regression.yml

```diff
@@ -29,10 +29,11 @@ jobs:
   regression:
     runs-on: ubuntu-latest
     strategy:
+      max-parallel: 4
       fail-fast: false
       matrix:
         include:
-          - { tool: klayout, part: drc, test: main }
```

Comment:
shouldn't we leave the `main` `drc` tests?

Reply:
@proppy No main is the entire rule deck. No need as we are going to granular testing. table per table is better for CI.

Reply:
I see, is `switch` also included in `main`?

Reply:
No, switch testing is not unit testing of rules. It basically tests if you turn certain switches on would the rule deck run and not exit with an exception or not. It's a totally different type of testing.